Sources/TensorFlow/Core/LazyTensorOperation.swift

```diff
@@ -95,7 +95,7 @@ class LazyTensorHandle: _AnyTensorHandle {
     var lazyTensorOperation: LazyTensorOperation? {
         switch handle {
         case .symbolic(let op, _, _): return op
-        case .concrete(_): return nil
+        case .concrete(_, _): return nil
```

Comment:
You can also just write
```suggestion
        case .concrete: return nil
```

Reply:
Even better! Done.